index.html

```diff
@@ -46,20 +47,177 @@ <h1 class="headline-title">I'm Uzair.<br>Glad to see you!</h1>
+                    <a href="#" class="btn">See Project</a>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="work-preview">
+                    <img class="project-img" src="resources/images/Snapshoot Portfolio 3.png" alt="Project 1">
+                </div>
+                <div class="work-detail">
+                    <h2 class="work-title">Tonic</h2>
+                    <ul class="span-grid">
+                        <li class="color">CANOPY</li>
+                        <img class="dot" src="resources/icon/dot.png" alt="dot">
+                        <li >Back End Dev</li>
+                        <img class="dot" src="resources/icon/dot.png" alt="dot">
+                        <li>2015</li>
+                    </ul>
+                    <p class="work-details">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+                    <ul class="work-tech">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <a href="#" class="btn">See Project</a>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="work-preview">
+                    <img class="project-img" src="resources/images/Snapshoot Portfolio 4.png" alt="Project 1">
+                </div>
+                <div class="work-detail">
+                    <h2 class="work-title">Multi-Post Stories</h2>
+                    <ul class="span-grid">
+                        <li class="color">CANOPY</li>
+                        <img class="dot" src="resources/icon/dot.png" alt="dot">
+                        <li >Back End Dev</li>
+                        <img class="dot" src="resources/icon/dot.png" alt="dot">
+                        <li>2015</li>
+                    </ul>
+                    <p class="work-details">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+                    <ul class="work-tech">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <a href="#" class="btn">See Project</a>
+                </div>
+            </div>
+        </section>
+        <!-- About Me Section-->
+        <section class="about-me">
+            <div>
+                <h2 class="headline-title">About Myself</h2>
+                <div class="article-2 about-me">
+                    <p class="headline-content">
+                        I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+                    </p>
+                    <a class="social-title" href="index.html">LET'S CONNECT</a>
```

Comment:
- [ ] I would suggest you to use heading tag like `h2` or `h3` for `LET'S CONNECT` heading as it is not suppose to be a link its just a heading for the following social media tags and heading tag would be semantically more appropriate.